app/main.py

```diff
@@ -1,3 +1,57 @@
 class Distance:
-    # Write your code here
-    pass
+
+    def __init__(self, km):
+        self.km = km
+
+    def __str__(self):
+        print(f"Distance: {self.km} kilometers")
+
+    def __repr__(self):
+        return f"Distance(km={self.km})"
+
+    def __add__(self, other):
+        if isinstance(other, Distance):
+            return Distance(
+                km=self.km + other.km
+            )
+        return Distance(
+            km=self.km + other
+        )
+
+    def __iadd__(self, other):
+        if isinstance(other, Distance):
+            self.km += other.km
+            return self
+        self.km += other
```

Comment:
Should add some check on variable 'other' type

Reply:
It is optional for this task